spec/functional/bootstrap_download_spec.rb

```diff
@@ -184,11 +171,6 @@ def run_download_scenario
       @template_output = sample_data('win_template_rendered_with_bootstrap_install_command.txt')
     end
 
-    it "bootstrap_install_command option is rendered in the windows-chef-client-msi.erb template", :chef_lt_12_5_only => true do
-      expect(bootstrap.send(:render_template,@template_input)).to eq(
-        @template_output)
-    end
-
     context "when running chef-client 12.5.0 or greater", :chef_gte_12_5_only => true, :chef_lt_14_only => true  do
```

Comment:
Can we just verify this one or make changes to work from gte 13 and remove gte_12_5_only ?

Reply:
Thank you @Vasu1105, Updated and removed extra template file and filter used here.